bubblesub/model.py

```diff
@@ -167,25 +167,32 @@ def __len__(self) -> int:
         """
         return len(self._items)
 
+    def __bool__(self) -> bool:
+        """Return if the collection is empty.
+
+        :return: True, if the collection is empty, False otherwise
+        """
+        return bool(self._items)
```

Comment:
Out of scope but because it offers `__iter__`, I'd also override `__reversed__` for completeness